x-pack/plugins/observability/public/pages/overview/overview.stories.tsx

```diff
@@ -328,7 +328,7 @@ storiesOf('app/Overview', module)
     registerDataHandler({
       appName: 'infra_metrics',
       fetchData: fetchMetricsData,
-      hasData: async () => true,
+      hasData: async () => ({ hasData: true, indices: 'metric-*' }),
     });
```

Comment:
This is repeated verbatim with identical args a few times, worth extracting as a function?